Library/Homebrew/test/formula_installer_bottle_spec.rb

```diff
@@ -49,26 +51,46 @@ def temporarily_install_bottle(formula)
     expect(formula).not_to be_latest_version_installed
   end
 
-  specify "basic bottle install" do
-    allow(DevelopmentTools).to receive(:installed?).and_return(false)
-    Homebrew.install_args.parse(["testball_bottle"])
-    temporarily_install_bottle(TestballBottle.new) do |f|
-      # Copied directly from formula_installer_spec.rb
-      # as we expect the same behavior.
+  def test_basic_formula_setup(f)
+    # Test that things made it into the Keg
+    expect(f.bin).to be_a_directory
 
-      # Test that things made it into the Keg
-      expect(f.bin).to be_a_directory
+    expect(f.libexec).to be_a_directory
 
-      expect(f.libexec).to be_a_directory
+    expect(f.prefix/"main.c").not_to exist
 
-      expect(f.prefix/"main.c").not_to exist
+    # Test that things made it into the Cellar
+    keg = Keg.new f.prefix
+    keg.link
+
+    bin = HOMEBREW_PREFIX/"bin"
+    expect(bin).to be_a_directory
+
+    expect(f.libexec).to be_a_directory
+  end
 
-      # Test that things made it into the Cellar
-      keg = Keg.new f.prefix
-      keg.link
+  specify "basic bottle install" do
+    allow(DevelopmentTools).to receive(:installed?).and_return(false)
+    Homebrew.install_args.parse(["testball_bottle"])
+    temporarily_install_bottle(TestballBottle.new) do |f|
+      test_basic_formula_setup(f)
+    end
+  end
 
-      bin = HOMEBREW_PREFIX/"bin"
-      expect(bin).to be_a_directory
+  specify "basic bottle install with cellar information on sha256 line" do
+    allow(DevelopmentTools).to receive(:installed?).and_return(false)
+    Homebrew.install_args.parse(["testball_bottle_cellar"])
+    temporarily_install_bottle(TestballBottleCellar.new) do |f|
+      test_basic_formula_setup(f)
+
+      skip_relocation = if OS.linux?
+        # skip_relocation is always false on linux
+        BottleSpecification.new.skip_relocation?
+      else
+        true
+      end
```

Comment:
I was hoping this could be calling the same method on Mac or Linux but, given it's not, I think this would be fine:

```suggestion
      # skip_relocation is always false on Linux but can be true on macOS.
      # see: extend/os/linux/software_spec.rb
      skip_relocation = !OS.linux?
```